makefile

```diff
@@ -0,0 +1,59 @@
+# Makefile for building, shipping, and testing the container.
```

Comment:
Which yes, is not a great place to put it but this component blueprint can't really be tested without an example backend, so the whole building and testing outside of a simple `docker build` has been done in `test/` and `examples/`. Probably could use some improvements, which I was planning on tackling this Q as part of the general set of work to improve this.